exampleCustomAudioDevice/src/main/java/com/twilio/video/examples/examplecustomaudiodevice/FileAndMicAudioDevice.kt

```diff
@@ -314,28 +315,24 @@ class FileAndMicAudioDevice(private val context: Context) : AudioDevice {
 
     // Renderer helper methods
     @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
-    private fun writeOnLollipop(audioTrack: AudioTrack?, byteBuffer: ByteBuffer?, sizeInBytes: Int): Int {
+    private fun write(audioTrack: AudioTrack?, byteBuffer: ByteBuffer?, sizeInBytes: Int): Int {
```

Comment:
We can remove the `@RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)` attribute too, right?

Reply:
yeah good catch